setup.py

```diff
@@ -19,6 +19,12 @@
             "pick-install=pick_kernel.kernelspec:main",
         ]
     },
+    install_requires=[
```

Comment:
I usually prefer a requirements.txt file and loading it for ease of finding and updating these.

Reply:
Cool, I've shifted this over to match papermill's structure.